addons/mail/static/tests/tours/mail_composer_test_tour.js

```diff
@@ -264,3 +264,38 @@ registry.category("web_tour.tours").add("mail/static/tests/tours/mail_composer_t
         },
     ],
 });
+
+registry.category("web_tour.tours").add("mail_composer_attachment_removal_tour", {
+    steps: () => [
+
+    {
+        content: "click on send by email",
+        trigger: ".o_statusbar_buttons > button[name='action_quotation_send']",
+        run: "click"
+    },
+    {
+        content: "save a new layout",
+        trigger: ".o_technical_modal button[name='document_layout_save'], .o_field_widget[name='attachment_ids'] li > button",
+        run() {
+            document.querySelector(
+                ".o_technical_modal button[name='document_layout_save']"
+            )?.click();
+        }
+    },
+    {
+        content: "delete attachment",
+        trigger: ".o_field_widget[name='attachment_ids'] li > button",
+        run: "click"
```

Comment:
`li > button` is obscure on intention. I don't trust `content` in a sense because this acts like a comment, the selector should try as much as possible to match the semantics of end-users.

This button is a "X" on the ui. `.fa-times` is icon of "X". Using it in selector matches the intention of user to click on "X".

_(Out-of-scope, but this icon also has name `.fa-close`, it would have been more appropriate in code, don't change it in PR!)_